src/services/services.ts

```diff
@@ -3653,6 +3710,10 @@ namespace ts {
 
             let completionData = getCompletionData(fileName, position);
             if (!completionData) {
+                let entries = getJsDocCompletionEntries(fileName, position);

```

Comment:
``getCompletionData`` returns a list of ``symbol``, which seems to be an overkill for jsdoc tag names.